app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -82,6 +85,14 @@ class SubscriptionController @Inject()(
 
   val desAddressForm = new DesAddressForm(Logger, blacklistedPostCodes)
 
+  private val linkAccountForm: Form[LinkAccount] =
```

Comment:
we should have forms/models in models package
and optionally you rename "ans" & "lc" so its first glance readable, but its fine as is

Reply:
For future tech debt/discussion.